File: laravel/app/Models/Cliente.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Cliente extends Model
{
    use HasFactory;

    protected $fillable = [
        'nome',
        'cpf_cnpj',
        'telefone',
        'email',
        'endereco',
    ];

    /**
     * Get the vehicles for the client.
     */
    public function veiculos()
    {
        return $this->hasMany(Veiculo::class);
    }

    /**
     * Get the service orders for the client.
     */
    public function ordensServico()
    {
        return $this->hasMany(OrdemServico::class);
    }

    /**
     * Get the appointments for the client.
     */
    public function agendamentos()
    {
        return $this->hasMany(Agendamento::class);
    }

    /**
     * Get the quotes for the client.
     */
    public function orcamentos()
    {
        return $this->hasMany(Orcamento::class);
    }
}

File: Mobile/app/src/main/java/com/example/mobile/data/models/Cliente.kt
package com.example.mobile.data.models

import com.example.mobile.data.models.Veiculo
import com.google.gson.annotations.SerializedName

data class Cliente(
    val id: Int,
    val nome: String,
    val cpf: String,
    val telefone: String,
    val email: String,

    @SerializedName("veiculos")
    val veiculos: List<Veiculo>
)